Find a functionally equivalent code:com.android.connectivitymanagertest.functional.ConnectivityManagerMobileTest.verifyCellularConnection:COMMENT
<com.android.connectivitymanagertest.functional.ConnectivityManagerMobileTest: void verifyCellularConnection()>
Method Modifier: public      private     
Method Modifier: Yes         No          
Method Updated:  No          Yes         
Comment:// help function to verify 3G connection

@@ -1,5 +1,5 @@
 {
-    NetworkInfo extraNetInfo = mCM.getActiveNetworkInfo();
+    NetworkInfo extraNetInfo = mCm.getActiveNetworkInfo();
     assertEquals("network type is not MOBILE", ConnectivityManager.TYPE_MOBILE, extraNetInfo.getType());
     assertTrue("not connected to cellular network", extraNetInfo.isConnected());
 }

